crates/nu-parser/src/lite_parse.rs

```diff
@@ -55,6 +55,12 @@ enum BlockKind {
     SquareBracket,
 }
 
+#[cfg(target_family = "windows")]
+const ESCAPE_CHAR: char = '^';
```

Comment:
NOTE: I'm a bit surprised nushell decided to use native path delimiter on windows given that everything else is unified (e.g. all std. commands are builtins).

Reply:
Actually, this means we can't use `^` (traditional windows escape) because you use it for switch to external cmd.. hmm

Reply:
Definitely agree, especially given point (3) in https://github.com/nushell/nushell/pull/2126#discussion_r450474035, but I'm also not a Windows user.

@jonathandturner can probably comment more on that, since he does most of the Windows things 🙂